tests/test_geometry.py

```diff
@@ -765,37 +778,27 @@ def test_check_collisions_3():
     #      |   |
     #      |---|
     #
-    s_p = []
-    s_l = []
 
-    s_p.append(geometry.Coordinate(0, 0))
-    s_p.append(geometry.Coordinate(0, 2))
-    s_p.append(geometry.Coordinate(2, 2))
-    s_p.append(geometry.Coordinate(2, 0))
-
-    s_l.append(geometry.Line(s_p[0], s_p[1]))
-    s_l.append(geometry.Line(s_p[1], s_p[2]))
-    s_l.append(geometry.Line(s_p[2], s_p[3]))
-    s_l.append(geometry.Line(s_p[3], s_p[0]))
+    points_square = [
+        geometry.Coordinate(0, 0),
+        geometry.Coordinate(0, 2),
+        geometry.Coordinate(2, 2),
+        geometry.Coordinate(2, 0),
+    ]
 
     square = geometry.Region()
-    for entity in s_l:
-        square.add_entity(entity)
-
-    t_p = []
-    t_l = []
-
-    t_p.append(geometry.Coordinate(1, 2.2))
-    t_p.append(geometry.Coordinate(2.2, 1))
-    t_p.append(geometry.Coordinate(4, 4))
+    # create and add line entities to region from their respective points
+    add_line_entities_from_points(square, points_square)
```

Comment:
Not sure I like this function. Would prefer to have something like ```new_lines = create_lines_from_points()``` and append this to EntityList. ```square.entities += new_lines``` or ```square.add_entities(new_lines)```